docs/secret-rotation.md

```diff
@@ -0,0 +1,254 @@
+# Secret Rotations &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; **FluxCD**
```

Comment:
The formatting looks odd in different browser window sizes, let's just keep things as simple as possible.
```suggestion
# Secret Rotation
```